File: spec/java/src/io/kaitai/struct/spec/TestNavParentFalse2.java
package io.kaitai.struct.spec;

import io.kaitai.struct.testformats.NavParentFalse2;
import org.testng.annotations.Test;

import static org.testng.Assert.assertEquals;

public class TestNavParentFalse2 extends CommonSpec {
    @Test
    public void testNavParentFalse2() throws Exception {
        NavParentFalse2 r = NavParentFalse2.fromFile(SRC_DIR + "fixed_struct.bin");

        assertEquals(r.parentless().foo(), 80);
    }
}

File: spec/python/test_nav_parent_false2.py
import unittest

from nav_parent_false2 import NavParentFalse2

class TestNavParentFalse2(unittest.TestCase):
    def test_nav_parent_false2(self):
        r = NavParentFalse2.from_file("src/fixed_struct.bin")

        self.assertEqual(r.parentless.foo, 80)
